tools/build-docs.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+base_dir="$( cd "$(dirname "$0")/.." >/dev/null 2>&1 ; pwd -P )"
+readonly base_dir
```

Comment:
When I do this, Shellcheck complains:
```
Declare and assign separately to avoid masking return values.shellcheck(SC2155)
```
See https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
Ok.